transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerInvoke.java

```diff
@@ -202,12 +202,16 @@ public void execute() {
     }
   }
 
-  private Holder<Boolean> checkQpsFlowControl(OperationMeta operationMeta) {
+  private boolean checkQpsFlowControl(OperationMeta operationMeta) {
     Holder<Boolean> qpsFlowControlReject = new Holder<>(false);
     @SuppressWarnings("deprecation")
     Handler providerQpsFlowControlHandler = operationMeta.getProviderQpsFlowControlHandler();
     if (null != providerQpsFlowControlHandler) {
       try {
+        Handler marKHandler = operationMeta.getMarkHandler();
```

Comment:
match & flowcontrol is different handler, and match should executed before flowcontrol. 

It's better to add an new method like 'processmatch' to deal match handler.

Reply:
has refactory